src/components/TodoComponents/TodoList.js

```diff
@@ -1,2 +1,17 @@
 // your components will all go in this `component` directory.
 // feel free to change this component.js into TodoList.js
+import React from 'react';
+function TodoList (props) {
+          return (
+              <div>
+                  {
+                      props.todo.map((todoObj) => {
+                          console.log(todoObj);
+                          return  (<div key="{todoObj}">{todoObj.task}</div>)
```

Comment:
To solve the id problem of the console.log you can try adding and id to the task